addons/point_of_sale/models/pos_order.py

```diff
@@ -153,10 +153,15 @@ def _process_order(self, order, draft, existing_order):
         if pos_order.to_invoice and pos_order.state == 'paid':
             pos_order._generate_pos_order_invoice()
 
-        if pos_session._is_capture_system_activated():
-            pos_session._remove_capture_content(order)
+        self._remove_capture_content(order)
         return pos_order.id
 
+    def _remove_capture_content(self, order_data):
```

Comment:
I'm not sure if it is a good idea to have the exact same function name than the one in the `pos.session`. It make it a bit confusing in my opinion and make it looks like there is recursivity. But I'm not sure how to rename it to make the differentiation clearer as the purpose is the same for both function, maybe `def _remove_session_captured_order` or something like this ?

Reply:
@lse-odoo I modified the code. Could you please check it again?